### PR TITLE
api, login

### PR DESCRIPTION
**Pull request by marijarahman**

### CODE REVIEW

**Review by NemanjaManot — APPROVED** [review]

File: src/actions/types.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
 export const GET_DATA = 'GET_DATA';
 export const GET_DATA_SUCCESS = 'GET_DATA_SUCCESS';
 export const GET_DATA_ERROR = 'GET_DATA_ERROR';
+
+export const LOGIN_ACTION_TYPES = {
+    TRY_LOGIN: 'TRY_LOGIN',
+    SET_ERROR: 'SET_ERROR'
+};
```
Comment (marko2606) [inline src/actions/types.js:8]: I like this.

File: src/components/LogIn/LogIn.js [hunk 2]
```diff
@@ -72,13 +98,37 @@ export default class LogIn extends React.Component {
...
+                        <Input
+                            placeholder="Username"
+                            onChange={this.handleUsernameChanged.bind(this)}
+                        />
+                        <Input
+                            placeholder="Password"
+                            onChange={this.handlePasswordChanged.bind(this)}
+                            value={this.state.password}
+                            secureTextEntry={true}
+                        />
```
Comment (marko2606) [inline src/components/LogIn/LogIn.js]: Why do you have value prop for password Input and not for username?

File: src/components/LogIn/LoginBtn.js [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
+    return (
+        <TouchableOpacity
+            style={logInButton}
+            onPress={props.onPress}
+        >
+            <View style={logInButtonView}>
+                <Text style={logInButtonText}>LOGIN</Text>
+            </View>
+        </TouchableOpacity>
+    )
```
Comment (marko2606) [inline src/components/LogIn/LoginBtn.js:17]: My fault. Thanks :)

File: src/components/LogIn/LogIn.js (hunk 2 above)
Comment (marijarahman) [reply src/components/LogIn/LogIn.js]: There is no need for password value property also, thanks

**Review by marko2606 — APPROVED** [review]

**Pull request merged**